### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by anajsana**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# How to contirbute
+
+## Repo structure
+
+The document table of contents are defined in the `toc.md` file and each of the sections show as different `.md` files from the repository:
+
+* Introduction
+* Contirbution to existing projects
```
Comment (misappi) [inline CONTRIBUTING.md:8]: There's a typo ("contirbution")

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* Starting Open Source projects
+* References
+
+## Types of contirbutions
+
+There are many ways people can contribute to the content of this repo:
+
+* Propose new sections
+* Add new content to existing sections
+* Fix inaccuracies or inconsintencies
```
Comment (misappi) [inline CONTRIBUTING.md:18]: Another type ("inconsintencies")

**Review by misappi — CHANGES_REQUESTED** [review]
Look good to me. I just found some typos. Thanks a lot.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+## Repo structure
+
+The document table of contents are defined in the `toc.md` file and each of the sections show as different `.md` files from the repository:
+
+* Introduction
+* Contirbution to existing projects
+* Starting Open Source projects
+* References
+
+## Types of contirbutions
```
Comment (cornelius) [inline CONTRIBUTING.md:12]:
```suggestion
## Types of contributions
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+There are many ways people can contribute to the content of this repo:
+
+* Propose new sections
+* Add new content to existing sections
+* Fix inaccuracies or inconsintencies
+* Add new references
+* Translate content into other languages
```
Comment (cornelius) [inline CONTRIBUTING.md:20]: I think we would want to reach a "first release" state before we ask for translations.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+> To get started, please open an [issue](https://github.com/todogroup/outbound-oss/issues)
```
Comment (cornelius) [inline CONTRIBUTING.md:22]: I would also ask people to open pull requests, if they have concrete changes to suggest.

**Review by cornelius — COMMENTED** [review]
Generally looks good to me. Some small comments inline.

File: CONTRIBUTING.md (hunk 4 above)
Comment (misappi) [reply CONTRIBUTING.md:20]: Makes sense. Agree.

Comment (anajsana) [reply CONTRIBUTING.md]: OK! I've deleted that part in a new commit 👍

File: CONTRIBUTING.md (hunk 5 above)
Comment (anajsana) [reply CONTRIBUTING.md]: Done!

**Review by anajsana — COMMENTED** [review]
I've just added some commits to make the requested changes 👍 thanks a lot for the review!

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# How to contirbute
```
Comment (cornelius) [inline CONTRIBUTING.md:1]: Typo: "contirbute"

**Pull request merged**